src/com/content-event/event-graded-game.js

```diff
@@ -0,0 +1,15 @@
+import {h, Component}					from 'preact/preact';
+import UIButtonLink from 'com/ui/button/button-link';
+
+export default class GradedGame extends Component {
```

Comment:
Technically it should be `Item` not `Game`, but we should go deeper.

It should call-out that it's Content, a child of the Event, and belonging to the user by naming it `ContentEventMyGrades`

Reply:
Kind of fixed by other means making it `GradedItem` as a component in the same file as the `MyGrades`-page since it is only used there and very simple at the moment. Then we can solve where it should be later.